ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/Init.hs

```diff
@@ -231,13 +231,13 @@ initStartingWith tracer cfg stream initDb = do
         let replayed' :: Word64
             !replayed' = replayed + 1
 
-            events :: [LedgerEvent blk]
-            events = inspectLedger
+            evs :: [LedgerEvent blk]
```

Comment:
~Why is this renaming necessary?~ Ah, I see, because of the `events` field in `ApplyOpts`. I suggest to rename `ApplyOpts.events` to `ApplyOpts.computeEvents`/`ApplyOpts.requestEvents` and keep `events` for these.